Frends.Salesforce.ExecuteQuery/CHANGELOG.md

```diff
@@ -3,3 +3,7 @@
 ## [1.0.0] - 2022-03-07
 ### Added
 - Initial implementation of Frends.Salesforce.ExecuteQuery
+
+## [1.0.1] - 2022-05-27
+### Added
+- Structure update in result object

```

Comment:
Changelog needs to be more informative to the user, so he knows what has changed between versions. This should be something like "fix for issue with referencing result-object in other elements".